### PR TITLE
Remove DangerJS and update dependencies

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
* Removes DangerJS, functionality replaced with probot-changelog app
* Updates fs-extra dependency

### CODE REVIEW

**Comment by emilyrohrbough** [comment]
Can you trigger a changelog 'failure'?

**Comment by bjankord** [comment]
@emilyrohrbough Without a changelog entry and a change to one of the files specified in the directories in [.github/changelog.yml](https://github.com/cerner/terra-aggregate-translations/blob/master/.github/changelog.yml#L3) the changelog status will stay at `Pending — changelog entry missing` and the status check will not pass until a changelog update is added.

Screen Shot 2019-06-07 at 9 27 18 AM

File: lib/aggregate-messages.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// trigger changelog failure
```
Comment (emilyrohrbough) [inline lib/aggregate-messages.js]: This guy needs clean up

**Review by emilyrohrbough — APPROVED** [review]

File: lib/aggregate-messages.js (hunk 1 above)
Comment (bjankord) [reply lib/aggregate-messages.js]: Cleaned up in this commit: https://github.com/cerner/terra-aggregate-translations/pull/6/commits/aad64411639b7eea971a457cc92a0efb6f8d6745

**Review by StephenEsser — APPROVED** [review]

**Review by ryanthemanuel — APPROVED** [review]

**Pull request merged**